app/src/main/java/edu/nextstep/camp/calculator/MainActivity.kt

```diff
@@ -1,46 +1,79 @@
 package edu.nextstep.camp.calculator
 
 import android.os.Bundle
+import android.widget.Toast
 import androidx.appcompat.app.AppCompatActivity
 import edu.nextstep.camp.calculator.databinding.ActivityMainBinding
+import edu.nextstep.camp.calculator.domain.Calculator
+
 class MainActivity : AppCompatActivity() {
     private lateinit var binding: ActivityMainBinding
 
+    private var expression = ""
```

Comment:
```suggestion
    private var expression = ""
```
expression을 String 타입으로 해서 멤버 변수로 사용해주고 계시네요.
값을 변경하는 멤버 변수를 사용하게 되면, clickOperator, clickOperand 등 함수의 호출 순서에 따라 영향을 많이 받게 돼요.
Expression을 관리하는 클래스를 만들어서, 해당 클래스에서 수식을 관리하도록 변경해보면 어떨까요?

Reply:
현 상황에서 호출순서에따라 영향을 받는다는 말이 어떤느낌인지 잘 이해가되지 않습니다 ㅠㅜ 하지만 확실히 클래스로 만들어 관리를하면 가독성,재사용성과 관리측면에서 확실히 좋을것같습니다!! 수정하겠습니다~

Reply:
이 부분을 제가 답변을 안드렸네요 🙏 
호출 순서에 따라 영향을 받는다는 부분은, 가령 clickEquals 함수를 호출할 때 clickOperator와 clickOperand를 어떻게 호출하느냐에 따라 결괏값이 다르다는 의미였습니다.
그리고 수식과 관련된 요구사항이 추가가 되면, 관련 코드가 MainActivity에 더 많아지게 되서 점점 유지보수가 어려워집니다 😅